include/message_filters/subscriber.h

```diff
@@ -161,61 +136,80 @@ struct message_type <M, false>
 template <typename M>
 using message_type_t = typename message_type<M>::type;
 
-template<class M, class NodeType = rclcpp::Node>
+template<class M>
 class Subscriber 
-: public SubscriberBase<NodeType>
+: public SubscriberBase
 , public SimpleFilter<message_type_t<M>>
 {
 public:
-  typedef std::shared_ptr<NodeType> NodePtr;
   typedef message_type_t<M> MessageType;
   typedef MessageEvent<MessageType const> EventType;
 
   /**
-   * \brief Constructor
+   * \brief Constructor for rclcpp::Node / rclcpp_lifecycle::LifecycleNode
    *
    * See the rclcpp::Node::subscribe() variants for more information on the parameters
```

Comment:
perhaps link the rclcpp::create_subscription here, as we want to get rid of specific Node implementations

Reply:
Removed the sentence